src/js/utils/can-autoplay.js

```diff
@@ -25,21 +25,42 @@
+    }
+    return URL.createObjectURL(videoBlob);
```

Comment:
Instead of all the `try catch` below, what if we check if the method exists, rather than recovering from failure?
```js
const method = (window.URL && window.URL.createObjectURL) ||
    (window.webkitURL && window.webkitURL.createObjectURL) // etc.
```

At the end of the day, third-party scripts might override not only `URL`, but also `Blob`, etc.